Cargo.toml

```diff
@@ -15,6 +15,7 @@ datafusion = "41"
 jiter = "0.5"
 paste = "1"
 log = "0.4"
+arrow-cast = "52.0.0"
```

Comment:
Not sure if it's okay to add this dep or if it will cause version conflicts...

Reply:
If we have to that's ok, I was trying to avoid it if possible to make upgrades easier.

Are you sure you can't get this from datafusion?

Reply:
removed